features/03_addproduct.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Add a product to the cart
```

Comment:
this feature should be named
03_addproducttocart.feature 
or 
03_add_product_to_cart.feature